package.json

```diff
@@ -4,6 +4,6 @@
         "gulp-autoprefixer": "^3.1.0",
         "gulp-rename": "^1.2.2",
         "gulp-rev": "^7.0.0",
-        "gulp-sass": "^2.3.1"
+        "gulp-ruby-sass": "^2.0.6"
```

Comment:
I was having trouble installing `gulp-sass` but I can try again. I suppose `gulp-sass` does make it easier to pull down new versions since you don't have to work with both gem and npm.